src/server/qgsfcgiserverresponse.cpp

```diff
@@ -177,7 +176,7 @@ void QgsSocketMonitoringThread::run()
 
   if ( mShouldStop.load() )
   {
-    QgsDebugMsgLevel( QStringLiteral( "FCGIServer::run %1: socket monitoring quits normally." ).arg( threadId ), 2 );
+    QgsMessageLog::logMessage( QStringLiteral( "FCGIServer::run %1: socket monitoring quits normally." ).arg( threadId ), QStringLiteral( "FCGIServer" ), Qgis::MessageLevel::Info );
```

Comment:
This message is gonna be printed every 300ms, it's gonna flood the log, this should stay a debug message IMHO

Reply:
oh... you're right...